java/node/src/test/java/com/codeheadsystems/dstore/node/BaseSQLTest.java

```diff
@@ -39,14 +41,15 @@ public abstract class BaseSQLTest extends BaseMetricTest {
   private static final Logger log = LoggerFactory.getLogger(BaseSQLTest.class);
 
   protected SqlEngine sqlEngine;
-  private DataSourceManager dataSourceManager;
+  protected DataSourceManager dataSourceManager;
   private DatabaseInitializationEngine databaseInitializationEngine;
-  @Mock private DatabaseConnectionEngine databaseConnectionEngine;
+  @Mock
+  private DatabaseConnectionEngine databaseConnectionEngine;
+  protected final String url = "jdbc:hsqldb:mem:" + getClass().getSimpleName() + ":" + UUID.randomUUID();
```

Comment:
We want this set per test, not once for the class.

Reply:
oh, the URL? sure, I can revert that part, was from an earlier iteration